modules/ROOT/pages/api-led-test.adoc

```diff
@@ -113,7 +113,7 @@ image:payload-response.png[Breakpoint at APIkit Router showing error message in
 
```

Comment:
Let's also update 4.2.10 image so the APIKit router icon is updated

Reply:
Image has been updated